cmd/cspc-operator/app/handler.go

```diff
@@ -122,6 +121,18 @@ func (c *Controller) syncCSPC(cspcGot *apis.CStorPoolCluster) error {
 		return nil
 	}
 
+	// cleaning up CSPI resources in case of removing poolSpec from CSPC
+	// or manual CSPI deletion
+	if cspcGot.DeletionTimestamp.IsZero() {
```

Comment:
this need to be '!'

Reply:
That would be a CSPC deletion case which is covered in handleCSPC deletion. This is CSPI removal case.

Reply:
That would be a CSPC deletion case which is handled in handleCSPCDeletion function. When a CSPI is removed then CSPC will not have a DeletionTimeStamp.

Reply:
ok.. it have to be IsZero itself.. correct